src/mixins/borrow/cooksV2.js

```diff
@@ -101,11 +105,19 @@ export default {
 
       let buyToken = pool.collateralToken.address;
       if (this.isGlp) {
-        const leverageResp = await getGlpLevData(this.signer, pool, amount, 42161, slipage)
+        const leverageResp = await getGlpLevData(
+          this.signer,
+          pool,
+          amount,
+          42161,
```

Comment:
hardcoded chainid

Reply:
there are no code changes here, just that the formatter has updated the view - just like with the previous one, we will update cook.js when we get completely away from the old config